src/main/java/com/leverx/employeestat/rest/dto/converter/WorkConverter.java

```diff
@@ -0,0 +1,57 @@
+package com.leverx.employeestat.rest.dto.converter;
+
+import com.leverx.employeestat.rest.dto.WorkDTO;
+import com.leverx.employeestat.rest.entity.Work;
+import com.leverx.employeestat.rest.entity.WorkId;
+import com.leverx.employeestat.rest.exception.EntityConversionException;
+import com.leverx.employeestat.rest.repository.EmployeeRepository;
+import com.leverx.employeestat.rest.repository.ProjectRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.Column;
+import java.util.UUID;
+
+@Component
+public class WorkConverter {
+
+    private final EmployeeRepository employeeRepository;
+
+    private final ProjectRepository projectRepository;
```

Comment:
converter shouldn't contain the repoes calls. Normally, service interacts with database and pass all necessary info for the converters and converter do conversion between models